src/components/Mascot.tsx

```
@@ -0,0 +1,1465 @@
+      if (walkInterval.current) { cancelAnimationFrame(walkInterval.current as unknown as number); clearInterval(walkInterval.current) }
+      if (frenzyTimeout.current) clearTimeout(frenzyTimeout.current)
+
+      // FRENZY MODE — 60 seconds of excited running + quotes + money
+      setFrenzy(true)
+      setState('frenzy')
+      setCrabOnBox(false)
+      onBoxRef.current = false
+
+      // Excited fast walk — bounce across screen
+      let frenzyDir: 'left' | 'right' = Math.random() > 0.5 ? 'right' : 'left'
+      let frenzyTarget = frenzyDir === 'right' ? 85 : 10
```

ESLint

[error] 'frenzyTarget' is never reassigned. Use 'const' instead.

(prefer-const)